### ISSUE TITLE
Picker Toggle UX Issue

### ISSUE THREAD

**Issue by felixfaire** [cocopon/tweakpane#89]
Hi, great work on this library, its been a joy to use so far.

Just wanted to mention a minor UX issue that when toggling a colour picker or point2D panel the intuition is that you can close the panel by clicking on the button that opened it (the colour square for colour or + sign for the 2D panel). I'm using this library to expose some design options to a client and want them to have as smooth an experience as possible. It took me a few clicks to realise how to close these panels and sure they will too as it is not obvious that you have to click outside the panel (but not on the button that opened it).

As a side note it would be great to be able to increase the width of the default panel from the js side. At the moment my variable names are all bunched up due to the fixed (quite small) size. Any suggestions for where I can change this in style etc would be helpful.

Thanks again for the great work!

**Comment by cocopon** [comment]
Thank you!

### minor UX issue
You mean that...:

1. Click a color square (or 2d panel) button -> picker opened
2. Click the button again -> ...

Expected: picker should be closed
Actual: picker is closed and then re-opened?

Sorry if it's wrong, I'm not a native English speaker and I'm not sure if that is correct...

### be able to increase the width of the default panel from the js side
I think it's style matter and it should be handled by CSS. If you omit `container` option, Tweakpane create a default wrapper element and its width is 256px. You can specify a container element of any width.

```js
const pane = new Tweakpane({
  container: document.getElementById('wideContainer'),
});
```

**Comment by felixfaire** [comment]
That is exactly what i meant yes (and includes the other modal popups like point2D). And thanks for the css tips.

**Comment by cocopon** [comment]
Created detailed and single topic issue: #90

**Comment by cocopon** [comment]
Fixed in #91. Works on most modern browsers (Chrome, Firefox, and Edge).

Still doesn't work on Safari for technical reason...:
https://github.com/cocopon/tweakpane/blob/81a05a70e630c093de980dad78bc4a804ec76670/src/main/js/misc/dom-util.ts#L95

Please wait for the next release.
